Info about MFA (#6380)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

\* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.

## October 2024
 

> - **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced on all users who log in with username and password credentials. 

Reviewer: ```suggestion
- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced when anyone signs in with username and password credentials. 
```

## October 2024
 

> - **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-acces/mfa) is now enforced on all users who log in with username and password credentials. 

Author: ```suggestion
- **Behavior change:** [Multi-factor authentication](/docs/cloud/manage-access/mfa) is now enforced on all users who log in with username and password credentials. 
```
